src/receivers/AwsLambdaReceiver.ts

```diff
@@ -0,0 +1,245 @@
+import { Logger, ConsoleLogger, LogLevel } from '@slack/logger';
+import querystring from 'querystring';
+import crypto from 'crypto';
+import tsscmp from 'tsscmp';
+import App from '../App';
+import { Receiver, ReceiverEvent } from '../types/receiver';
+import { ReceiverMultipleAckError } from '../errors';
+
+export interface AwsEvent {
+  body: string | null;
+  headers: any;
+  multiValueHeaders: any;
+  httpMethod: string;
+  isBase64Encoded: boolean;
+  path: string;
+  pathParameters: any | null;
+  queryStringParameters: any | null;
+  multiValueQueryStringParameters: any | null;
+  stageVariables: any | null;
+  requestContext: any;
+  resource: string;
+}
+
+export type AwsCallback = (error?: Error | string | null, result?: any) => void;
+
+export interface AwsResponse {
+  statusCode: number;
+  headers?: {
+    [header: string]: boolean | number | string;
+  };
+  multiValueHeaders?: {
+    [header: string]: Array<boolean | number | string>;
+  };
+  body: string;
+  isBase64Encoded?: boolean;
+}
+
+export type AwsHandler = (event: AwsEvent, context: any, callback: AwsCallback) => Promise<AwsResponse>;
+
+export interface AwsLambdaReceiverOptions {
+  signingSecret: string;
+  logger?: Logger;
+  logLevel?: LogLevel;
+}
+
+export interface AwsLambdaReceiverInstallerOptions {
```

Comment:
@seratch I don't see you using `installPath` or `redirectUriPath`. Did you add them because you wanted to implement OAuth but decided against it? For now, the recommendation is if you want OAuth, use `ExpressReceiver`

Reply:
Oh, thanks for pointing this out! Yes, I was going to implement OAuth flow too but, as I mentioned in the issue, `@slack/oauth` module is not for this receiver at this point. I will remove this interface and add some comments why OAuth is not yet supported before merging this PR.

Reply:
Resolved this.